samples/bluetooth/mesh/ble_peripheral_lbs_coex/README.rst

```diff
@@ -1,7 +1,7 @@
 .. _bluetooth_ble_peripheral_lbs_coex:
 
-Bluetooth: Mesh and peripheral coexistence
-##########################################
+Bluetooth mesh: Peripheral coexistence
```

Comment:
Now I would rename this as: 
```suggestion
Bluetooth mesh: Coexistence with other LE services
```

The thing is "Peripheral" role already exist in all mesh nodes. All mesh nodes, with the default configuration are already, GATT peripherals by definition. However, all that magic happens inside the stack.

What this sample is showing, how to add "another" BLE service to the existing mesh node.

Reply:
The way you explain it, it makes sense to me :) That said, could you then have a look at the Overview section of this sample and see if more text should be tweaked? Because now I am not sure myself.